src/common/form/Form.js

```diff
@@ -43,13 +41,20 @@ class Form extends React.Component<Props> {
     }
   };
 
-  render(): React.Element<*> {
+  render(): ?React.Element<*> {
```

Comment:
nit: this should be `React.Node` now. They recently changed this

Reply:
are you sure? React.Node contains React.Element inside itself. So, with Node it becomes less strict. If we know what particularly returns from the render function we can do React.Element<typeof View> or something instead View.
Ofc, since we don't use the type anyways, we can do React.Node just for simplicity..or somehow disable eslint-rule for annotating render() return type and write nothing, like for functions which return undefined.